hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,7 @@
+RELEASE_TYPE: minor
+
+:func:`~hypothesis.strategies.text` now occasionally generates from a preselected list of strings which are likely to find bugs. These include ligatures, right-to-left and top-to-bottom text, emojis, emoji modifiers, strings like ``"Infinity"``, ``"None"``, and ``"FALSE"``, and other terrible things.
+
+This is especially useful when testing the full unicode range, where the search space is too large for uniform sampling to be very effective.
```

Comment:
I think many users will interpret this to mean that the "interesting" strings - I wouldn't use the word `terrible` - are constants and won't shrink as usual; we could add an explicit note that of course they do.  (which I believe is correct, since we're below the provider level, right?)

I'd also like to see a test that checks our shrinking works as expected; it could be that `minimal(<a fancy "The"> in string) in {set of fancy "The"s}` - not too improbable to find, easy to see how we found it (...until AST analysis, anyway) and also easy to confirm that it's a minimal rather than an exact input.